_posts/_2021-10-05-Launch-Announcement-1-1.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+authors: 
+  - henkle
+  - ahopp
```

Comment:
`ahopp` is the not the correct author id - `andhopp` is. https://github.com/opensearch-project/project-website/blob/main/_authors/andhopp.markdown